playground/backend/internal/setup_tools/life_cycle/life_cycle_setuper.go

```diff
@@ -39,6 +39,16 @@ func Setup(sdk pb.Sdk, code string, pipelineId uuid.UUID, workingDir string) (*f
 		return nil, err
 	}
 
+	// copy necessary files
+	switch sdk {
+	case pb.Sdk_SDK_GO:
+		err = lc.CopyFiles(workingDir, preparedModDir)
+		if err != nil {
+			logger.Errorf("%s: RunCode(): CopyFiles(): %s\n", pipelineId, err.Error())
+			return nil, err
```

Comment:
At this point, we have already created files/folders by `lc.CreateFolders()` method. So we need to delete them for each `err` case in this method.

Reply:
Done.